stories/topics/con-gcp-backup-process.adoc

```diff
@@ -18,7 +18,9 @@ The backup playbook requires an active {AAPonGCP} foundation deployment to be ru
 
 A bucket needs to be created in your project as restore information will be stored in that bucket.
 
-Only one backup per version is kept in the bucket, if multiple versions of the backup need to be retain then a new bucket must be created for the new version of the backup.
+The bucket can contain multiple backups from the same deployment or different ones. A backup will generate a directory named <prefix>-<deployment_name>-<timestamp> and a file <prefix>-<deployment_name>-<timestamp>.json. The directory contains the awx and pulp databases backup, the deployment configuration and the secrets. The json file contains information for the restore procedure.
```

Comment:
A backup generates a directory

and a file named <prefix....

Reply:
The file is postfix by json and doesn't so doesn't have the same name as the directory.

Reply:
ok got it you want me to add the workd "named" after the "file"